images/scripts/lib/build-deps.sh

```diff
@@ -49,7 +49,8 @@ case "$OS_VER" in
 esac
 
 # TEMP: cockpit needs python3-devel to select the default Python version
-EXTRA_DEPS="$EXTRA_DEPS python3-devel"
+# TEMP: asciidoctor needed for PR testing https://github.com/cockpit-project/cockpit/pull/21515
+EXTRA_DEPS="$EXTRA_DEPS python3-devel asciidoctor"
```

Comment:
Please don't drop the Python comment, we should clean this up but not in this PR :)